services/core/shared/core/transactionStatistics.js

```diff
@@ -260,10 +261,41 @@ const init = async historyLengthDays => {
 
 const updateTodayStats = async () => fetchTransactionsForPastNDays(1, true);
 
+const updateTransactionStatistics = async historyLengthDays => {
```

Comment:
Please consider updating the method name to reflect its exact utility.

```suggestion
const verifyTransactionStatistics = async historyLengthDays => {
```
or
```suggestion
const verifyAndUpdateTransactionStatistics = async historyLengthDays => {
```
or
```suggestion
const validateTransactionStatistics = async historyLengthDays => {
```